CONTRIBUTING.md

```diff
@@ -121,6 +121,16 @@ Before creating your pull request you have two options regarding what to do with
 
 If a pull request is received with a bewildering commit history that is difficult to reason with (e.g. 10+ commits containing obscure commit messages, previously committed code, old branch merges, etc), it will most likely be deemed too risky and therefore closed immediately.
 
+### Changesets
+
+Each Pull Request requires at least one changeset as part of the pull requests. Changesets will allow automatic deployments of code when they are merged to `master` or one of the feature branches.
+
+The general recommendation is to have only one changeset per pull requiest.
```

Comment:
If I understand correctly, we are recommending that PRs to master _always_ include a changeset while PRs to version releases don't have any. Maybe we should make this more clear.